pkg/processor/runtime/python/encode.go

```diff
@@ -35,15 +35,16 @@ func NewEventJSONEncoder(logger nuclio.Logger, writer io.Writer) *EventJSONEncod
 	return &EventJSONEncoder{logger, writer}
 }
 
-// Encode writes the JSON encoding of event to the stream, followed by a newline character.
+// Encode writes the JSON encoding of event to the stream, followed by a newline character
 func (je *EventJSONEncoder) Encode(event nuclio.Event) error {
-	je.logger.DebugWith("encoding", "event", event)
+	je.logger.DebugWith("Sending event to wrapper", "event", string(event.GetBody()))
```

Comment:
I'm not sure logging the body is a good idea. It might be big.

Reply:
You're right. But for now - we need all the information we can get to debug things